src/evaluator/from-flake.nix

```diff
@@ -0,0 +1,22 @@
+{ inputs
+, system
+, ...
```

Comment:
I would even go so far and make this signature finite. (non-variadic)

The reason is that it forces people to respect this interface and they won't be able to hack like hell (the root cause of why `nixpkgs` is too degraded for widespread adoption.

Reply:
nice argument, gonna fix it in next MR